dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/WorkflowVersionDAO.java

```diff
@@ -63,4 +93,45 @@ public List<WorkflowVersion> getTagsByWorkflowIdOrderedByLastModified(long workf
         query.setMaxResults(limit);
         return query.getResultList();
     }
+
+    private List<Predicate> processQuery(String sortCol, String sortOrder, CriteriaBuilder cb, CriteriaQuery query, Root<WorkflowVersion> version) {
+        List<Predicate> predicates = new ArrayList<>();
+
+        if (!Strings.isNullOrEmpty(sortCol)) {
+            if ("open".equalsIgnoreCase(sortCol)) {
+                if ("desc".equalsIgnoreCase(sortOrder)) {
+                    query.orderBy(cb.desc(version.get("versionMetadata").get("publicAccessibleTestParameterFile")), cb.desc(version.get("id")));
+                } else {
+                    query.orderBy(cb.asc(version.get("versionMetadata").get("publicAccessibleTestParameterFile")), cb.asc(version.get("id")));
```

Comment:
Nit-picky, and you don't have to change, but there's a lot of duplication throughout this method with column names and method invocations. It increases the odds of a typo (each time you type in a name, it is a new chance to spell it wrong, lol). Plus if we ever change anything in the future, it makes it easier to modify if it's only in one place.

You could introduce variables, constants, and/or add method(s) to reduce this.